src/services/userServices.js

```diff
@@ -23,7 +23,20 @@ export default class UserServices {
  * @returns {object} return the user's data
  */
   static async getUserByEmail(email) {
-      const data = await models.User.findOne({ where: { email, } });
-      return data;
+    const data = await models.User.findOne({ where: { email } });
+    return data;
+  }
+
+  /**
+   * @name LoginUser
```

Comment:
I think this method is not necessary.....my thoughts

Reply:
changed it